client/e2e/pages/EventsPage.ts

```diff
@@ -125,30 +136,24 @@ export class EventsPage {
     this.ticketQuantityOption = page.getByLabel('Ticket Quantity:');
     this.showingCard = page.getByTestId('showing-card');
     this.deleteShowingButton = page.getByRole('button', {name: 'Delete'});
+    this.activateEventchecker = page.getByLabel('controlled');
+    this.inactiveEventchecker = page.getByRole('button', { name: 'Inactive' });
+    this.cartButton = page.getByRole('button', { name: 'Cart' });
+    this.goToHomepageFromCart = page.getByRole('button', { name: 'back to Events' });
   }
 
   async clickLeftBar() {
     await this.leftBarEvent.click();
   }
 
-  /*
-  To simulate only clicking the edit button and do almost nothing.
-  This test is more geared toward testing the buttons.
-  */
-  async editEvents() {
-    await this.firstEvent.click();
-    await this.editEventInfo.click();
-    await this.editEventName.click();
-    await this.editEventDes.click();
-    await this.eventOption1.click();
-    await this.imageURL.click();
-    await this.editCancel.click();
-    await this.editAddShowing.click();
-    await this.editCancelShowing.click();
-    await this.editShowingId.click();
-    await this.cancelShowingId.click();
+  /**
+   * Switches the status of the event's activeness.
+   */
+  async activateEvent() {

```

Comment:
Either use this consistently and remove your add active event, and add inactive event, or use your functions exclusively and refactor the other tests. I recommend using this function only and removing the other two functions

Reply:
Yup, please use this.